common/utils/src/main/resources/error/error-conditions.json

```
@@ -3580,6 +3580,23 @@
     ],
     "sqlState" : "42805"
   },
+  "PARQUET_CONVERSION_FAILURE" : {
+    "message" : [
+      "Unable to create a Parquet converter for the data type <dataType> whose Parquet type is <parquetType>."
+    ],
+    "subClass": {
+      "WITHOUT_DECIMAL_METADATA" : {
+        "message" : [
+          " Please read this column/field as Spark BINARY type."
```

The gap is not needed because we merge parent's and chid's messages via a space.
```suggestion
          "Please read this column/field as Spark BINARY type."
```